### PR TITLE
some doc clarifications/corrections in array and array_slice

### PR DESCRIPTION
**Pull request by nilsbecker**

### CODE REVIEW

File: src/core/CCArray_slice.mli [hunk 1]
```diff
@@ -83,23 +83,26 @@ val reverse_in_place : 'a t -> unit
 val sorted : ('a -> 'a -> int) -> 'a t -> 'a array
 (** [sorted cmp a] makes a copy of [a] and sorts it with [cmp].
     @since 1.0 *)
-
+
```
Comment (c-cube) [inline src/core/CCArray_slice.mli]: there's a trailing space here.

**Comment by c-cube** [comment]
Thanks for the changes. I'll merge them when you remove the trailng space.

**Comment by nilsbecker** [comment]
ok, ist there anything else to do? i copied the mli sections unchanged form array to array_slice, i think this was the case also before...

**Comment by c-cube** [comment]
It's fine, thanks!

**Pull request merged**